wiki/Reporting_Bad_Behaviour/de.md

```diff
@@ -0,0 +1,106 @@
+| Sexueller Missbrauch oder extremes Fehlverhalten | Gehe zu [Missbrauch melden](/wiki/Reporting_Bad_Behaviour/Abuse) |
+
+Für alles andere, siehe die Anweisungen weiter unten.
+
+## Chat im Spiel
+
+### Der !report-Befehl
+
+![](img/report-command.jpg "Beispiel von der Benutzung des !report-Befehls")
+
+Fehlverhalten im Chat ist der häufigste Verstoß. Um das zu melden, benutze den [`!report`](https://osu.ppy.sh/community/forums/topics/34843) Befehl, vorzugsweise in dem Channel, in dem der Verstöß stattfand. Fühle dich nicht gestresst: wenn deine Nachricht mit `!report` und einem Leerzeichen beginnt, **dann ist es nur für Moderatoren sichtbar** und niemand sonst im Chat. Du kannst auch die Meldefunktion via dem [Spiel-Overlay](#in-game-overlay) wie unten beschrieben nutzen, wenn du immer noch Bedenken hast.
+
+Die Befehlssyntax hat zwei Formen:
+
+```
+!report Benutzername
+!report Benutzername (freie Beschreibung folgt)
+```
+
+Wenn sich Leerzeichen im Benutzernamen befinden, ersetze diese mit Unterstrichen:
+
+```
+!report Schlechter_Benutzer spamt in #osu
+```
+
+### Spiel-Overlay
+
+![](img/report-user-1.png "Benutzeraktion mit dem Report-Knopf")
+
+![](img/report-user-2.png "Meldedialog im Spiel")
+
+Diese Option funktioniert, wenn der schlechte Benutzer in Sicht liegt:
+
+1. Klicke seinen Benutzernamen und drücke den roten Knopf auf dem steht `Report User...`
+2. Wähle die Kategorie und füge Details hinzu (optional).
+
+## Webseite
+
+### Profilseite
+
+![](img/report-user-profile.png "Der Meldeknopf auf einer Profilseite")
+
+![](img/report-user-web.png "Meldedialog auf der Webseite")
+
+1. Drücke den Knopf mit den drei vertikalen Punkten und wähle `Report`.
+2. Wähle die Kategorie und füge Details hinzu (optional).
+
+### Forum
+
+![](img/report-user-forum.png "Der Meldeknopf in einem Forumbeitrag")
+
+1. Fahre mit der Maus über den Beitrag.
+2. Drücke in einer Reihe an Knöpfen, die in der rechten oberen Ecke des Beitrags erscheinen, auf den Knopf mit einer Warnmeldung.
+3. Wähle die Kategorie und füge Details hinzu (optional).
+
+### Kommentarsektion
+
+![](img/report-user-comment.png "Der Meldeknopf in einem Kommentar auf der Webseite")
+
+1. Klicke auf den `Report`-Link.
+2. Wähle die Kategorie und füge Details hinzu (optional).
+
+### Beatmap
+
+![](img/report-beatmap.png "Der Meldeknopf auf einer Beatmapseite")
+
+1. Drücke die drei vertikalen Punkte und wähle `Report`.
+2. Wähle die Kategorie und füge Details hinzu (optional).
+
+### Beatmap-Diskussion
+
+![](img/report-user-discussion.png "Der Meldeknopf in einem Diskussionsbeitrag")
+
+1. Klicke den `Report`-Link unter dem Diskussionsbeitrag.
+2. Wähle die Kategorie und füge Details hinzu (optional).
+
+### Scoreliste
+
+![](img/report-score-user.png "Der Meldeknopf an einem Top-Score auf einer Benutzerprofilseite")
+
+#### Benutzerprofil
+
+1. Scrolle zu der `Ranks`-Sektion oder `Historical` für aktuelle Plays und gehe auf den Score.
```

Comment:
```suggestion
1. Scrolle zu der `Ränge`-Sektion oder `Historisch` für aktuelle Plays und gehe auf den Score.
```

Reply:
This is now the question which I added in the PR's description. This text refers to the image above. Since the image is in English, the text in the sentence should too, in my opinion.

Reply:
If we change this, every other mention of button texts and stuff would also need to be changed.

Reply:
oh sorry, i overlooked that one. If the image is in english we should also keep the references to it in the text in english. Alternatively could make new screenshots in german, but that can be done at a later time maybe. For now, while the images are english, it's fine to keep the text english aswell then.